src/JsonCppLib/JsonCppLib.vcxproj

```diff
@@ -73,18 +94,34 @@
     <PlatformToolset>v142</PlatformToolset>
     <SpectreMitigation>Spectre</SpectreMitigation>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Fuzzing|x64'" Label="Configuration">
+    <ConfigurationType>StaticLibrary</ConfigurationType>
+    <UseDebugLibraries>false</UseDebugLibraries>
+    <PlatformToolset>v142</PlatformToolset>
+    <EnableASAN>true</EnableASAN>

```

Comment:
><EnableASAN>true</EnableASAN> [](http://example.com/codeflow?start=4&length=29)

EnableASAN for the other 3 fuzzing configurations?

Reply:
Going to remove them; I think they got auto added by VS.

Reply:
Then let's remove those configurations in CommonCore.vcxproj too